app/src/app/manual/staff/applications/page.tsx

```diff
@@ -65,6 +76,37 @@ export default function ApplicationsPage() {
       (a.motivation || "").length > 140
         ? `${a.motivation?.slice(0, 140)}…`
         : a.motivation || "",
+    // Human-friendly vote text for table
+    myVote: a.myVote ? (a.myVote === "APPROVED" ? "Approved" : "Rejected") : "Not voted",
+    // Actions: delete button for CODING_ADMIN
+    actions: (
+      <div>
+        {me && canDeleteStaffApplication(me.role) && (
+          <button
+            disabled={deleteMutation.isPending}
+            className={`text-red-600 hover:underline ${
+              deleteMutation.isPending ? "opacity-50 cursor-not-allowed" : ""
+            }`}
+            onClick={async (e) => {
+              e.preventDefault();
+              if (deleteMutation.isPending) return;
+              const ok = window.confirm("Delete this application? This cannot be undone.");
```

Comment:
We have a @Confirm component, which we'd usually use for confirm dialogs. Also there is a @Button component which we usually use for actions in table. See other places in the codebase for reference :)